src/main/kotlin/com/exactpro/th2/sim/template/rule/KotlinFIXRule.kt

```diff
@@ -44,15 +44,17 @@ class KotlinFIXRule(field: Map<String, Any?>) : MessageCompareRule() {
         private val TrdMatchId = AtomicInteger(0)
 
         private val buyOrdersAndIds: Queue<Pair<ParsedMessage, Int>> = LinkedList()
```

Comment:
Could we move state from static to class aria ?

Reply:
I suppose we could move order queues, but id counters should be in static context because we can have more than one instance of rule active, and we need to have globally unique ids for all orders/execution reports/trades in all rules.